src/app/src/user/controllers/UserDeleteCtrl.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+function UserListCtrl($scope, $state, UserRepository, $modal) {
+    var vm = this;
+    var viewPath = 'packages/gzero/admin/views/user/directives/';
+    // Delete modal
+    vm.deleteModal = {
+        /**
+         * Function initiates the AngularStrap modal
+         *
+         * @param title translatable title of modal
+         * @param message translatable messages of modal
+         */
+        initModal: function(title, message) {
+            var self = this;
+            self.modal = $modal({
+                scope: $scope,
+                title: title,
+                content: message,
+                template: viewPath + 'userDeleteModal.tpl.html',
+                show: true,
+                placement: 'center'
+            });
+        },
+
+        /**
+         * Function shows the AngularStrap modal
+         *
+         * @param userId content id to be removed, it is saved in the scope
+         */
+        showModal: function(userId) {
+            var self = this;
+            vm.userId = userId;
+            self.initModal('PLEASE_CONFIRM', 'DELETE_USER_QUESTION');
+        },
+
+        /**
+         * Function close the modal
+         *
+         */
+        closeModal: function() {
+            var self = this;
+            self.modal.hide();
+        },
+        /**
+         * Function performs the RestAngular DELETE action for user id in scope
+         *
+         */
+        deleteContent: function() {
```

Comment:
I think that this function name should be 'deleteUser' instead of 'deleteContent'